cdap-ui/server/aggregator.js

```diff
@@ -149,7 +159,7 @@ Aggregator.prototype.pushConfiguration = function(resource) {
 
   this.connection.write(
     JSON.stringify({
-      resource: resource,
+      resource: this.stripAuthHeaderInProxyMode(resource),
```

Comment:
just curious, is this to prevent the auth token and user id being logged in the client side?

Reply:
Yeah. If auth is in proxy mode then the client doesn't need to know the auth token and the other headers related to authorization since it is the proxy that is adding the headers.